Documentation/fault-output.rst

```diff
@@ -99,35 +105,24 @@ The output is parametrized by the Pickpoint namelist, example:
   PPFileName = 'fault_receivers.dat'
   /
 
-printtimeinterval
-~~~~~~~~~~~~~~~~~
-
-The output is generated every printtimeinterval (local) time step. Using
-this output with local time-stepping may result in differently sampled
-receiver files.
+**printtimeinterval** determines how frequently the output is generated — every **printtimeinterval** (local) time step. Please note that using this output with local time-stepping may result in differently sampled receiver files.
 
 .. _ioutputmask-1:
 
 iOutputMask
 ~~~~~~~~~~~
 
-same as for ParaView output.
-
-
-seissolxdmf
-~~~~~~~~~~~
-
-SeisSol paraview files (XDMF/Hdf5 or XDMF/binary files, describing the fault outputs and the free-surface/volume wavefield) can also be read using our python module `seissolxdmf <https://pypi.org/project/seissolxdmf/>`__.
+This is the same as for the ParaView output.
 
 Additional Ascii output
 -----------------------
 
-The rupture front can be outputted at every gauss points by enabling RF_output_on.
+You can output the rupture front at every Gauss point by enabling **RF_output_on** in the DynamicRupture namelist:
```

Comment:
This has been dropped out when porting DR to cpp.
Parameters currently read can be find in this file:
https://github.com/SeisSol/SeisSol/blob/master/src/DynamicRupture/Parameters.h

Reply:
Sounds good!